content/en/content-management/organization/index.md

```diff
@@ -186,11 +186,11 @@ example.com/posts/new-post/
 
 ### `section`
 
-`section` is determined by a content's location on disk and *cannot* be specified in the front matter. See [sections][] for more information.
+`section` is determined by a content item's location on disk and *cannot* be specified in the front matter. See [sections][] for more information.
 
 ### `type`
 
-A content's `type` is also determined by its location on disk but, unlike `section`, it *can* be specified in the front matter. See [types][]. This can come in especially handy when you want a piece of content to render using a different layout. In the following example, you can create a layout at `layouts/new/mylayout.html` that Hugo will use to render this piece of content, even in the midst of many other posts.
+A content item's `type` is also determined by its location on disk but, unlike `section`, it *can* be specified in the front matter. See [types][]. This can come in especially handy when you want a piece of content to render using a different layout. In the following example, you can create a layout at `layouts/new/mylayout.html` that Hugo will use to render this piece of content, even in the midst of many other posts.
```

Comment:
```suggestion
A content item's `type` is also determined by its location on disk but, unlike `section`, it *can* be specified in the front matter. See [types]. This can come in especially handy when you want a piece of content to render using a different layout. In the following example, you can create a layout at `layouts/new/mylayout.html` that Hugo will use to render this piece of content, even in the midst of many other posts.
```